[2022-01-26 17:58:20,147] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.check manual__2022-01-26T17:58:18.347781+00:00 [queued]>
[2022-01-26 17:58:20,206] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.check manual__2022-01-26T17:58:18.347781+00:00 [queued]>
[2022-01-26 17:58:20,206] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-26 17:58:20,207] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-26 17:58:20,207] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-26 17:58:20,263] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): check> on 2022-01-26 17:58:18.347781+00:00
[2022-01-26 17:58:20,277] {standard_task_runner.py:52} INFO - Started process 6510 to run task
[2022-01-26 17:58:20,286] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_data', 'check', 'manual__2022-01-26T17:58:18.347781+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/dag_01.py', '--cfg-path', '/tmp/tmpifpt9lyt', '--error-file', '/tmp/tmpa_dlytgg']
[2022-01-26 17:58:20,287] {standard_task_runner.py:77} INFO - Job 113: Subtask check
[2022-01-26 17:58:20,431] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_data.check manual__2022-01-26T17:58:18.347781+00:00 [running]> on host 223810fcea28
[2022-01-26 17:58:20,566] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=check
AIRFLOW_CTX_EXECUTION_DATE=2022-01-26T17:58:18.347781+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-26T17:58:18.347781+00:00
[2022-01-26 17:58:20,570] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-26 17:58:20,571] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo -------------------------- EMPIEZA ------------------------']
[2022-01-26 17:58:20,597] {subprocess.py:85} INFO - Output:
[2022-01-26 17:58:20,603] {subprocess.py:89} INFO - -------------------------- EMPIEZA ------------------------
[2022-01-26 17:58:20,604] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-26 17:58:20,653] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=check, execution_date=20220126T175818, start_date=20220126T175820, end_date=20220126T175820
[2022-01-26 17:58:20,708] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-26 17:58:20,775] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
